.github/workflows/msh3.yml

```diff
@@ -64,10 +64,10 @@ jobs:
     - run: make V=1
       name: 'make'
 
-    - run: make V=1 examples
-      name: 'make examples'
+    - run: make V=1 -C tests
```

Comment:
Why skip building examples here?

Reply:
It was not intentional to do that change, but then again we could probably just as well skip them in this build as they are built in many others.

Reply:
maybe we should consider only building the example in a single specific example job and remove the building of them from all others?

Reply:
Good idea, but we should at least cover all platforms and architectures as we had issues on some in the past with the examples.